cmd/agent/common/system_probe_config_helper.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build process, !windows
+
+package common
+
+import (
+	"os"
+	"path"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	proc_config "github.com/DataDog/datadog-agent/pkg/process/config"
+)
+
+// SetupSystemProbeConfig reads the system-probe.yaml into the global config object
+func SetupSystemProbeConfig(sysProbeConfFilePath string) error {
+	// Open the system-probe.yaml file if it's in a custom location
+	if sysProbeConfFilePath != "" {
+		// If file is not set directly assume we need to add /system-probe.yaml
+		if !strings.HasSuffix(sysProbeConfFilePath, ".yaml") {
+			sysProbeConfFilePath = path.Join(sysProbeConfFilePath, "/system-probe.yaml")
```

Comment:
we don't need the extra `/` before `system-probe.yaml`

Also could we make `system-probe.yaml` a var/constant in the file ?